packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -33,11 +43,34 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
     onToggleChange (event: Event) {
         const target = event?.target as HTMLInputElement;
         this.isChecked = target.checked;
-        this.dispatchEvent(new CustomEvent(EVENT_TOGGLE_SWITCH_CHANGED, { detail: this.isChecked }));
+        const changedEvent = new CustomEvent(ON_TOGGLE_SWITCH_CHANGED_EVENT, { detail: this.isChecked });
+        this.dispatchEvent(changedEvent);
```

Comment:
for some reason, defining the custom event and dispatching it in the same line creates two events in the react wrapper.

<img width="1280" alt="Screenshot 2023-08-13 at 23 29 19" src="https://github.com/justeattakeaway/pie/assets/28605803/a439774d-91cd-4ba2-b896-0a58e24a27ad">

Reply:
It might sound like a dumb question... Are those events the same and why do we have both?

Reply:
yep, and the first one is wrong and shouldn't be added. 
when you inline the custom event and dispatch it as it was done previously, the first event is created, which shouldn't happen.

Reply:
Got it. I thought that was in runtime! 😨